.travis.yml

```diff
@@ -9,8 +9,10 @@ python:
 - "3.4"
 - "3.5"
 - "3.6"
+- "nightly"
```

Comment:
Want to replace "nightly" -> "3.7" but there is issue about Travis and python 3.7: https://github.com/travis-ci/travis-ci/issues/9815

Reply:
You can add a job with specific settings for that

Reply:
Nevermind. I did this.

Reply:
Nightly points to 3.8-dev now.